docs/adrs/0003-support-aggregations-on-publisher-name.md

```diff
@@ -0,0 +1,98 @@
+| |   | PublicationInformation  | Elsevier BV |
+| |   | PublicationInformation  | null |
+| |   | PublicationInformation | ACM|Creativity and Cognition |
+| |   |  PublicationInformation | Cambridge, Mass. : Alfred P. Sloan School of Management, Massachusetts Institute of Technology |
+| |   | PublicationInformation  | Elsevier |
+| |   | PublicationInformation  | Wiley |
+| ArchivesSpace | XML select array of Strings matching element `dc:publisher`  | PublicationInformation  | Massachusetts Institute of Technology. Libraries. Department of Distinctive Collections (note: this appears to be effectively a static string for all of our ASpace records) |
+| MIT GIS |   | PublicationInformation |  |
+| OGM GIS |   | PublicationInformation  |  |
+
+### Proposed mappings (Option 1)
+
+Create a new object `PublicationInfo` and map data into it. Most sources just use `Publication.name` but Alma uses additional fields.
+
+```json
+{
+  "publicationInfo": [
+    {
+      "name": "String",
+      "location": "String",
+      "date": "String (ideally dates would be dates... but that may be out of scope for now)"
+    }
+  ]
+}
+```
+
+Notes: we should consider whether this really needs to be an array. 260 and 264 are repeatable fields in MARC, but whether they are regularly used in that way or if we could instead pick the "first" one that shows up and get a simpler model without losing any real world functionality would be worth investigating. For now it is modeled as an array of `PublicationInfo` assuming that is appropriate.
+
+GraphQL would collapse the new object and continue to serve it as the deprecated field `PublicationInformation` until we are confident it is no longer being used.
+
+| Source | Mappings | TIMDEX Field | Notes |
+| --- | --- | --- | --- |
+| Alma | [MARC 260$a](https://www.loc.gov/marc/bibliographic/bd260.html) | PublicationInfo.Location | |
+| Alma | [MARC 264$a](https://www.loc.gov/marc/bibliographic/bd264.html) | PublicationInfo.Location | |
+| Alma | [MARC 260$b](https://www.loc.gov/marc/bibliographic/bd260.html) | PublicationInfo.Name | |
+| Alma | [MARC 264$b](https://www.loc.gov/marc/bibliographic/bd264.html) | PublicationInfo.Name | |
+| Alma | [MARC 260$c](https://www.loc.gov/marc/bibliographic/bd260.html) | PublicationInfo.Date | |
+| Alma | [MARC 264$c](https://www.loc.gov/marc/bibliographic/bd264.html) | PublicationInfo.Date | |
+| Alma | [MARC 260$d](https://www.loc.gov/marc/bibliographic/bd260.html) | Invalid field. Don't map | |
+| Alma | [MARC 260$e](https://www.loc.gov/marc/bibliographic/bd260.html) | Don't map | |
+| Alma | [MARC 260$c](https://www.loc.gov/marc/bibliographic/bd260.html) | Don't map | |
+| DSpace | Keep logic | Publication.Name | note: consider future work to normalize in source or during transform |
+| ASpace | Keep logic | PublicationInfo.Name | |
+| GIS MIT | | PublicationInfo.Name | |
+| GIS OGM | | PublicationInfo.Name | |
+
+### Proposed mappings (Option 2)
```

Comment:
@JPrevost For Option 2, would it also require only choosing the _first_ occurrence of the MARC 260 and 264 fields? 🤔

Reply:
Good question! I believe similar to the discussion in Option 1, the fields are repeatable but it's unclear how frequently they are actually repeatable. I'd suggest we may want to look at our data to decide if this should be an Array of Strings or a single String. I suspect an Array of Strings will be where we end up but I'd hate to make that decision based on suspicion rather than data.

Reply:
Just to tease this out a bit more... if:

- `publisher` is a multivalued string field
- and any other publisher information gets written to other fields like `dates` or `locations` ([see comment below](https://github.com/MITLibraries/transmogrifier/pull/132/files#r1506065717))

wouldn't this mean, for Option 2, that we could accomodate both MARC 260 and 264?  Stated more generally, that Option 2 supports multiple publishers by virtue of the new `publisher` field being multivalued and all other values mapped to other, multivalued fields?

Reply:
Correct. If we just make `publisher` an array of Strings, we can support any source that currently or in the future has multiple values for all aspects of this (dates, names, locations).

If we make `publisher` a single string, we could still support multiple Dates/Locations if that is the only thing that is truly multivalued in our sources. That may be a bit hard to tease out if that is all we truly need or if we should just accept that supporting multivalued is better because we just aren't sure.

Reply:
Awesome, thanks for the peek into the data. This is extremely helpful context. I feel like I have enough info to add some additional context to the Options and propose a Decision (option 2 seems popular with the top level array for publisher names and everything else broken into Location and Dates).